### ISSUE TITLE
The cookie expiration time "" is not valid

### ISSUE THREAD

**Issue by yasirrose** [Codeception/Codeception#3980]
#### What are you trying to achieve?
see text on site title
#### What do you get instead?

> Provide console output if related. Use `-vvv` mode for more details.
C:\xampp\htdocs\FamousSmokeN>php codecept.phar run -vvv
Codeception PHP Testing Framework v2.1.8
Powered by PHPUnit 5.3.2 by Sebastian Bergmann and contributors.

←[1mAcceptance Tests (1) ←[22m--------------------------------------------------
--------------------
Modules: ←[32mPhpBrowser, \Helper\Acceptance←[39m
--------------------------------------------------------------------------------
-----------
←[35;1mFamousSmokeCept←[39;22m
Scenario:
* I am on page "/"
←[37;41m ERROR ←[39;49m

--------------------------------------------------------------------------------
-----------

←[1mFunctional Tests (0) ←[22m------------------------
Modules: ←[32m\Helper\Functional, WebDriver←[39m
---------------------------------------------
---------------------------------------------

←[1mUnit Tests (0) ←[22m------------------------------
Modules: ←[32mAsserts, \Helper\Unit←[39m
---------------------------------------------
---------------------------------------------

Time: 4.24 seconds, Memory: 8.00Mb

There was 1 error:

---------
1) ←[1mFamousSmokeCept←[22m (tests\acceptance\\FamousSmokeCept.php)

←[37;41m
  ←[39;49m
←[37;41m  [UnexpectedValueException] The cookie expiration time "" is not valid.
  ←[39;49m
←[37;41m
  ←[39;49m
←[33m
Scenario Steps:
←[39m
←[1m 1. $I->amOnPage("/")←[22m

phar://C:/xampp/htdocs/FamousSmokeN/codecept.phar/vendor/symfony/browser-kit/Coo
kie.php:195
phar://C:/xampp/htdocs/FamousSmokeN/codecept.phar/vendor/symfony/browser-kit/Coo
kieJar.php:149
phar://C:/xampp/htdocs/FamousSmokeN/codecept.phar/vendor/symfony/browser-kit/Coo
kieJar.php:164
phar://C:/xampp/htdocs/FamousSmokeN/codecept.phar/vendor/symfony/browser-kit/Cli
ent.php:320
phar://C:/xampp/htdocs/FamousSmokeN/codecept.phar/src/Codeception/Lib/InnerBrows
er.php:158
phar://C:/xampp/htdocs/FamousSmokeN/codecept.phar/src/Codeception/Lib/InnerBrows
er.php:201
phar://C:/xampp/htdocs/FamousSmokeN/codecept.phar/src/Codeception/Lib/InnerBrows
er.php:245
phar://C:/xampp/htdocs/FamousSmokeN/codecept.phar/src/Codeception/Step.php:218
phar://C:/xampp/htdocs/FamousSmokeN/codecept.phar/src/Codeception/TestCase/Share
d/Actor.php:99
phar://C:/xampp/htdocs/FamousSmokeN/codecept.phar/src/Codeception/Scenario.php:1
01
C:\xampp\htdocs\FamousSmokeN\tests\_support\_generated\WebGuyActions.php:267
C:\xampp\htdocs\FamousSmokeN\tests\acceptance\FamousSmokeCept.php:7
phar://C:/xampp/htdocs/FamousSmokeN/codecept.phar/src/Codeception/TestCase/Cept.
php:70
phar://C:/xampp/htdocs/FamousSmokeN/codecept.phar/vendor/phpunit/phpunit/src/Fra
mework/TestCase.php:985
phar://C:/xampp/htdocs/FamousSmokeN/codecept.phar/vendor/phpunit/phpunit/src/Fra
mework/TestCase.php:845
phar://C:/xampp/htdocs/FamousSmokeN/codecept.phar/vendor/phpunit/phpunit/src/Fra
mework/TestResult.php:685
phar://C:/xampp/htdocs/FamousSmokeN/codecept.phar/vendor/phpunit/phpunit/src/Fra
mework/TestCase.php:801
phar://C:/xampp/htdocs/FamousSmokeN/codecept.phar/vendor/phpunit/phpunit/src/Fra
mework/TestSuite.php:747
phar://C:/xampp/htdocs/FamousSmokeN/codecept.phar/src/Codeception/PHPUnit/Runner
.php:89
phar://C:/xampp/htdocs/FamousSmokeN/codecept.phar/src/Codeception/SuiteManager.p
hp:153
phar://C:/xampp/htdocs/FamousSmokeN/codecept.phar/src/Codeception/Codecept.php:2
03
phar://C:/xampp/htdocs/FamousSmokeN/codecept.phar/src/Codeception/Codecept.php:1
72
phar://C:/xampp/htdocs/FamousSmokeN/codecept.phar/src/Codeception/Command/Run.ph
p:262
phar://C:/xampp/htdocs/FamousSmokeN/codecept.phar/src/Codeception/Command/Run.ph
p:189
phar://C:/xampp/htdocs/FamousSmokeN/codecept.phar/vendor/symfony/console/Command
/Command.php:256
phar://C:/xampp/htdocs/FamousSmokeN/codecept.phar/vendor/symfony/console/Applica
tion.php:791
phar://C:/xampp/htdocs/FamousSmokeN/codecept.phar/vendor/symfony/console/Applica
tion.php:186
phar://C:/xampp/htdocs/FamousSmokeN/codecept.phar/vendor/symfony/console/Applica
tion.php:117
phar://C:/xampp/htdocs/FamousSmokeN/codecept.phar/codecept:34
C:\xampp\htdocs\FamousSmokeN\codecept.phar:7

←[37;41mFAILURES!←[0m
←[37;41mTests: 1←[0m←[37;41m, Assertions: 0←[0m←[37;41m, Errors: 1←[0m←[37;41m.←
[0m

C:\xampp\htdocs\FamousSmokeN>php codecept.phar run --version
Codeception version 2.1.8

C:\xampp\htdocs\FamousSmokeN>
```bash
# paste output here
```
> Provide test source code if related

```php
// paste test
```
### Details

* Codeception version: 2.1.8
* PHP Version: 7.0.13
* Operating System:
* Installation type: Composer 
* List of installed packages 

 {
    "require": {
        "codeception/codeception": "2.2",
      
        "browserstack/browserstack-local": "dev-master"
    },
    "scripts": {
        "clean": "rm tests/_output/*; killall -9 BrowserStackLocal;",
        "debug-on": "export XDEBUG_CONFIG=idekey=session_name",
        "debug-off": "unset  XDEBUG_CONFIG",
        "local": "composer clean; vendor/bin/codecept run --env local",
        "remote": "composer clean; vendor/bin/codecept run --env remote"
    },
    "autoload": {
        "classmap": ["lib/"]
    },
    "config": {
        "bin-dir": "bin"
    }    
}

* Suite configuration:

```yml
actor: Tester
paths:
    tests: tests
    log: tests/_output
    data: tests/_data
    support: tests/_support
    envs: tests/_envs
settings:
    bootstrap: _bootstrap.php
    colors: true
    memory_limit: 1024M
extensions:
    enabled:
        - Codeception\Extension\RunFailed
modules:
    enabled:
       - Asserts:
       - WebDriver:
          #- url: https://ryanalin.famous-smoke.com
    config:
        Db:
            dsn: ''
            user: ''
            password: ''
            dump: tests/_data/dump.sql
        PhpBrowser:
             url: https://ryanalin.famous-smoke.com
        WebDriver:
            url: https://origin-staging.famous-smoke.com/
            browser: phantomjs
            capabilities:
                takesScreenshot: true
                phantomjs.cli.args: ["--ignore-ssl-errors=true"]
                phantomjs.page.settings.userAgent: 'Mozilla/5.0 (Windows NT 6.1; Win64; x64; rv:16.0) Gecko/20121026 Firefox/16.0; captcha_bypass_fam0us'
            headers:
                Accept: 'text/html,application/xhtml+xml,application/xml;q=0.9,*/*;q=0.8'
                Accept-Language: 'zh-CN,zh;q=0.8,en-US;q=0.5,en;q=0.3'
                User-Agent: 'Mozilla/5.0 (Windows NT 10.0; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/54.0.2840.99 Safari/537.36'
                Connection: 'keep-alive'
        \BrowserStackWebDriver:
          host: 'hub-cloud.browserstack.com'
          port: 80
          browser: Firefox
          # url:     https://ryanalin.famous-smoke.com
          url: https://origin-staging.famous-smoke.com/
          capabilities:
            os: 'Windows'
            os_version: 'XP'
            'browserstack.user': 'andrew1133'
            'browserstack.key' : 'n4UsPjUQWoB7fsExknx8'
            'browserstack.debug': false
            'browserstack.local': false
            'acceptSslCerts': false
            'build': 'codeception-local-str'
            'name': 'codeception-local-str'
        gherkin:
            contexts:
                default:
                    - AcceptanceTester
                    - CheckoutSteps

**Comment by Naktibalda** [comment]
Duplicate of #2301

Symfony BrowserKit was updated to ignore invalid cookie expiration dates [2 days ago](https://github.com/symfony/symfony/commit/a6c05e5fb677cc3c8ffa07f86652366ac0060b3e).

Please wait for the next release and upgrade.

**Comment by Naktibalda** [comment]
Released as v3.2.3, v2.8.17, v2.7.24
